File: JavaNgArch/src/projectbase/mvc/BaseViewModel.java


package projectbase.mvc;
import projectbase.mvc.result.JavaScriptSerializer;
    public class BaseViewModel
    {
        public String ToJsonString()
        {
            return (new JavaScriptSerializer()).Serialize(this);
        }
        public String ToJsonString(Object data)
        {
            return (new JavaScriptSerializer()).Serialize(data);
        }
    }

File: NetNgArch/src/ProjectBase/Web.Mvc/BaseViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web.Mvc;
using System.Web.Script.Serialization;

namespace ProjectBase.Web.Mvc
{
    public class BaseViewModel
    {
        public string ToJsonString()
        {
            return (new JavaScriptSerializer()).Serialize(this);
        }
        public string ToJsonString(object data)
        {
            return (new JavaScriptSerializer()).Serialize(data);
        }
    }

  

}
